Dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.11-slim
+
+# Set working directory
+WORKDIR .
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)